src/parser/jobs/pct/modules/Gauge.tsx

```diff
@@ -240,17 +250,81 @@ export class Gauge extends CoreGauge {
 
 		this.addEventHook(playerFilter.type('action').action(this.data.actions.COMET_IN_BLACK.id), this.onBlackPaintSpender)
 
-		this.addEventHook(playerFilter.type('action').action(this.data.actions.SUBTRACTIVE_PALLETTE.id), this.onSubtractivePalette)
+		this.addEventHook(playerFilter.type('action').action(this.data.actions.SUBTRACTIVE_PALETTE.id), this.onSubtractivePalette)
 
 		const canvasActions = Array.from(this.canvasModifiers.keys())
 		this.addEventHook(playerFilter.type('action').action(oneOf(canvasActions)), this.onCanvasModifier)
 
 		this.addEventHook(playerFilter.type('action').action(oneOf(this.depictionModifiers)), this.onDepictionModifier)
 
-		this.addEventHook(playerFilter.type('action').action(oneOf(this.portraitModifiers)), this.onPortraitModifier)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.portraitGenerators)), this.onPortraitGenerator)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.portraitSpenders)), () => this.portraitGauge.reset())
 
 		// Default assume the player painted before the pull
 		this.canvasGauge.set([CREATURE_MOTIF, WEAPON_MOTIF, LANDSCAPE_MOTIF])
+
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onComplete() {
+		// Suggestion to not overcap the palette gauge
+		const lostSubtractives = Math.floor(this.paletteGauge.overCap / SUBTRACTIVE_COST)
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SUBTRACTIVE_PALETTE.icon,
+			value: lostSubtractives,
+			content: <Trans id="pct.gauge.suggestions.lost-subtractives.content">Overcapping the Palette gauge will cause you to lose uses of <DataLink action="SUBTRACTIVE_PALETTE"/> over the course of the fight, leading to lost potency. Make sure to use your gauge before it overcaps.</Trans>,
+			why: <Trans id="pct.gauge.suggestions.lost-subtractives.why">You lose <Plural value={lostSubtractives} one="# use" other="# uses" /> of <DataLink showIcon={false} action="SUBTRACTIVE_PALETTE" /> due to overcapping the Palette gauge.</Trans>,
+			tiers: {
+				1: SEVERITY.MEDIUM,
+				2: SEVERITY.MAJOR,
+			},
+		}))
+
+		// Suggestion to not end the fight with palette gauge remaining
+		const wastedSubtractives = Math.floor(this.paletteGauge.value / SUBTRACTIVE_COST)
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SUBTRACTIVE_PALETTE.icon,
+			value: wastedSubtractives,
+			content: <Trans id="pct.gauge.suggestions.wasted-subtractives.content">Ending the fight with Palette gauge remaining means you could have used another set of <DataLink action="SUBTRACTIVE_PALETTE" /> spells for additional damage.</Trans>,
```

Comment:
How "bursty" is palette gauge gain?  Does this get into the realm of a knowing your kill time suggestion?  Seems very non-typical to me to make a recommendation like this.

Reply:
The Subtractive spells are near double the potency of their additive equivalents, and I hazily thought I remembered other places where we suggest making use of leftover gauge but I'm definitely wrong there. I'm fine with ditching it.